mirdata/jams_utils.py

```diff
@@ -0,0 +1,147 @@
+# -*- coding: utf-8 -*-
+"""functions for converting mirdata annotations to jams format
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import jams
+
+def jams_converter(
+        beat_data=None,
+        chord_data=None,
+        note_data=None,
+        f0_data=None,
+        section_data=None,
+        multi_section_data=None,
+        key_data=None,
+        artist=None,
+        duration=None,
+        title=None,
+        annotator=None,
+        sandbox=None):
+
+    """Convert annotations to JAMS format.
+
+    Args:
+        beat_data (list or None):
+            A list of tuples of (BeatData, str), where str describes the annotation (e.g. beats_1).
+        chord_data (list or None):
+            A list of tuples of (ChordData, str), where str describes the annotation.
+        note_data (list or None):
+            A list of tuples of (NoteData, str), where str describes the annotation.
+        f0_data (list or None):
+            A list of tuples of (F0Data, str), where str describes the annotation.
+        section_data (list or None):
+            A list of tuples of (SectionData, str), where str describes the annotation.
+        key_data (list or None):
+            A list of tuples of (KeyData, str), where str describes the annotation.
+        artist (str or None):
+            A string describing the artist name.
+        duration (str or None):
+            A float describing the track duration in seconds.
+        title (str or None):
+            A string describing the track title.
+        annotator (dict or None):
+            A dict describing the annotator (e.g. {name: annotator_1}
+        sandbox (dict or None):
+            A dict with extra information related to the track.
+
+    """
+
+    jam = jams.JAMS()
+
+    # metadata
```

Comment:
maybe we can have metadata be a dictionary input, e.g.
{ 'artist': 'asdf', 'duration': 270, 'title': 'asdfasdf'}

Then when we create the jams object, we have logic that checks if `jam.file_metadata` has the dictionary key as an attribute (as you did before), and if not adds it to the sandbox.